stories/tsconfig.json

```diff
@@ -3,7 +3,6 @@
     "extends": "../tsconfig.json",
     "include": ["../src/typings.d.ts", "./*.ts", "./*.tsx"],
     "exclude": ["src/**/*.spec.ts"],
-    "files": [""],
```

Comment:
please explain this red line :)

Reply:
It was removed in next-7.1.0 (https://github.com/gooddata/gooddata-react-components/pull/995/files#diff-658ea170c7899b26a2c2cb52209cf161L6)
I think we should merge it to "next-7.1.0-upgrade-hc7" to avoid conflict in the future when we merge this branch back to "next-7.x.x"

Reply:
Removing this line to solve error when running storybook test on CI. I've wanted to delay rebasing next-7.1.0-upgrade-hc7 branch with next-7.1.0 branch. And will rebase after all PRs are merged to feature branch.
But anyway i'm going to rebase with next-7.1.0 then we don't need this change anymore.

Reply:
done